### PR TITLE
[Netmanager] Added category column to candidate column

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- git pull origin staging
- git checkout ft-category-filter
- npm run stage

#### What are the relevant tickets?

- [AN-221](https://airqoteam.atlassian.net/browse/AN-221?atlOrigin=eyJpIjoiNWFjMmI5MDQxYmU3NGM2YzhkMjBkMjNmMmE4MDlhZDgiLCJwIjoiaiJ9)

#### Screenshots
category-1
category-2
category-3

### CODE REVIEW

File: netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js [hunk 1]
```diff
@@ -104,184 +102,211 @@ const CandidatesTable = (props) => {
...
+            }
+          },
+          {
+            title: 'Email',
+            field: 'email'
+          },
+          {
+            title: 'Category',
+            field: 'category'
+          },
```
Comment (Codebmk) [inline netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:213]: Adde the new field here

Comment (Baalmart) [reply netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:213]: Awesome, thanks!

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @Codebmk

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Codebmk 
LGTM

**Pull request merged**